### PR TITLE
Show operation without status.operationStatus existing

### PR DESCRIPTION
**Pull request by dthomson25**

### CODE REVIEW

**Review by alexmt — CHANGES_REQUESTED** [review]
There is no need to introduce `syncing` field in state. 

Then user clicks sync button or requests any other operation it sets `operation` field in application CRD. When controller starts executing operation it moves info about requires operations and progress into  `status.operationState` and null out `operation` field. 

UI should show operation if either `operation` or `status.operationState` is set.

**Comment by dthomson25** [comment]
Thanks for the feedback! I pushed a chance that removed the syncing state

File: src/app/applications/components/application-status-panel/application-status-panel.tsx [hunk 1]
```diff
@@ -19,6 +19,17 @@ export const ApplicationStatusPanel = ({application, showOperation, showConditio
...
+    let appOperationState = application.status.operationState;
+    if (application.operation && application.status.operationState === undefined) {
+        appOperationState = {
+            phase:  models.OperationPhases.InProgress,
+            startedAt: new Date().toISOString(),
+            operation: {
+                sync: {},
+            } as models.Operation,
+        } as models.OperationState;
+        appOperationState.phase = models.OperationPhases.InProgress;
```
Comment (alexmt) [inline src/app/applications/components/application-status-panel/application-status-panel.tsx]: Looks like this line it not needed

**Review by alexmt — APPROVED** [review]
LGTM. Fix one comment before merging

**Pull request merged**